flink-libraries/flink-table/src/main/scala/org/apache/flink/table/validate/FunctionCatalog.scala

```diff
@@ -350,7 +350,16 @@ class BasicOperatorTable extends ReflectiveSqlOperatorTable {
     SqlStdOperatorTable.EXISTS,
     // EXTENSIONS
     EventTimeExtractor,
-    ProcTimeExtractor
+    ProcTimeExtractor,
+    SqlStdOperatorTable.TUMBLE,
+    SqlStdOperatorTable.TUMBLE_START,
```

Comment:
Do we already support the `START/END` functions? We should let them unsupported until they are implemented and tested.